### ISSUE TITLE
Migration from osgQt to osgQOpenGL

### ISSUE THREAD

**Issue by nim65s** [Gepetto/gepetto-viewer#146]
Hi,

In recent developments, the [osgQt project](https://github.com/openscenegraph/osgQt) removed everything related to osgQt (they just kept the name of the project) and switched to the new Qt5 preffered way and use QOpenGL:

https://github.com/openscenegraph/osgQt/pull/22

@jmirabel : could you evaluate the difficulty for us to move to the new interface ? Depending on the required work to make the switch, we can either do it, or I can finish my packaging with an older version of osgQt.

**Comment by jmirabel** [comment]
It is hard to evaluate the time it takes to migrate.

Following your initial plan, you intend to provide a version of osgQt in robotpkg, that suits the need of gepetto-viewer ? This will not be different even if we switch so I think you should continue what you were doing.

**Comment by nim65s** [comment]
Yes, I can continue on the initial plan, but we would need to either have some CMake magic that allows to use both osgQt or osgQOpenGL, or to synchronize well for the upgrade.
